opm/parser/eclipse/Units/Units.hpp

```diff
@@ -242,15 +242,6 @@ namespace Opm {
                 return q / unit;
             }
         } // namespace convert
-
-
-#ifndef HAS_ATTRIBUTE_UNUSED
```

Comment:
yes. @babrodtk please confirm, even though it doesn't matter any longer this block can definitely go.